language/tools/move-cli/src/sandbox/utils/mod.rs

```diff
@@ -35,7 +35,7 @@ pub use mode::*;
 pub use on_disk_state_view::*;
 pub use package::*;
 
-pub(crate) fn get_gas_status(gas_budget: Option<u64>) -> Result<GasStatus<'static>> {
+pub fn get_gas_status(gas_budget: Option<u64>) -> Result<GasStatus<'static>> {
```

Comment:
Do we want to keep this crate public still? I don't see a reason for it to be public/it being used anywhere.

Reply:
I think I ended up using it in some testing code I was writing. But I can revert the change to that PR